meson.build

```diff
@@ -167,3 +167,30 @@ if get_option('enable-nnstreamer-tensor-filter')
    nnstreamer_dep = dependency('nnstreamer', required: true)
    subdir('nnstreamer/tensor_filter')
 endif
+
+if get_option('enable-android')
+  ndk_build = find_program('ndk-build', required : true)
+  jni_root = meson.current_source_dir() / 'jni'
+  jni_build_root = meson.build_root() / 'jni'
+
+  ndk_args = {
+    'NDK_PROJECT_PATH': jni_root,
+    'APP_BUILD_SCRIPT': jni_root / 'Android.mk',
+    'NDK_APPLICATION_MK': jni_root / 'Application.mk',
+    'NDK_LIBS_OUT': jni_build_root / 'libs',
+    'NDK_OUT': jni_build_root / 'objs',
+  }
+  ndk_additional_flags = ['-j2']
```

Comment:
how about number of processors than just 2?

Reply:
Do you have a number? Or I'll do 8.

Reply:
I was thinking more like `grep -c '^processor' /proc/cpuinfo`

Reply:
Sound decent. Done